### PR TITLE
Fix disk/operation_time to report average operation time

### PR DESCRIPTION
**Pull request by quentinmit**
collectd reported average operation time; OT reports total operation time, so we need to introduce a new metric for average operation time and divide the existing metrics.

Before this can be merged we need to merge https://github.com/GoogleCloudPlatform/opentelemetry-operations-collector/pull/49 and update the submodule in this PR.

### CODE REVIEW

File: otel/conf.go [hunk 1]
```diff
@@ -166,6 +166,8 @@ windowsperfcounters/mssql_{{.MSSQLID}}:
           - system.network.dropped
           - system.filesystem.inodes.usage
           - system.paging.faults
+          - system.disk.operation_time
+          - system.processes.count
```
Comment (qingling128) [inline otel/conf.go:170]: Should this line be reflected in the other golden files as well?

Comment (quentinmit) [reply otel/conf.go:170]: Yes, it should. Forgot to regenerate the golden files again.

**Review by qingling128 — APPROVED** [review]
LGTM

File: otel/conf.go (hunk 1 above)
Comment (qingling128) [reply otel/conf.go:170]: The good thing is that our tests caught it as well, which is what they are designed for :D

**Pull request merged**